### PR TITLE
updating the version of earthpy w newest workshop data

### PR DESCRIPTION
**Pull request by lwasser**
ok @betatim  trying this out and will wait for it to build FIRST

### CODE REVIEW

**Comment by lwasser** [comment]
You know what, Tim… I just found this page

https://hub.docker.com/r/earthlab/earth-analytics-python-env/builds/

I see that each time I’m even updating the readme it’s rebuilding! Do you see that? I do see that it also takes a while. Am I doing something wrong? I think I get what is going on generally, maybe?

File: README.md [hunk 1]
```diff
@@ -88,3 +87,20 @@ view all images on your computer, type
...
+1. add the package that you wish to add to the environment in a fork or separate branch
+2. submit a PR
+
+If the update is to `earthpy`, you must specify the commit number that you wish to build `earthpy` against. This will
+ensure that the docker image automagically rebuilds using the latest version of `earthpy`
+
+like this: `- git+https://github.com/earthlab/earthpy.git@283683affac9e46b1690c7913ebd2621c82ba43a`
+
+This PR should kick off a rebuild of the docker image. You need to WAIT for this build to complete.
+When it is complete you will see a new tagged docker container, here:
```
Comment (betatim) [inline README.md]: I hadn't been thinking of this repo and its docker builds when I made my comment about creating a PR.

Unfortunately docker hub will only start building the image for this repository after you merge it. So I think this should read: 

```
You need to WAIT for this build to complete if you want to use the image as part of a user image for a JupyterHub. When it is complete you will see a new tagged docker container, here:
```

**Comment by betatim** [comment]
Not really an expert on docker hub, mostly just use it to push/pull images from there. It does seem like it builds a new image each time `master` changes.

It does take a fair amount of time though. Don't think there is much you can do about that :-/

File: README.md (hunk 1 above)
Comment (lwasser) [reply README.md]: Thank you...so i should add: @betatim  ? does that seem reasonable? i'll just forget as there are a lot of steps for me here!! but i'll figure it out. 

```
You need to 
1. merge the PR submitted and then 
2. **WAIT** for this build to complete if you want to use the image as part of a 
user image for a JupyterHub. When it is complete you will see a new tagged docker 
container, here:
```

**Comment by betatim** [comment]
I would use the following workflow:
* make a PR with changes
* think about it
* merge the PR
* check&wait till dockerhub has built the image for the merging of the PR
* (go and update hub-ops with the newly tagged image)

**Pull request merged**